src/recordlinker/models/algorithm.py

```diff
@@ -54,7 +54,7 @@ class AlgorithmPass(Base):
     algorithm_id: orm.Mapped[int] = orm.mapped_column(schema.ForeignKey("algorithm.id"))
     algorithm: orm.Mapped["Algorithm"] = orm.relationship(back_populates="passes")
     blocking_keys: orm.Mapped[list[int]] = orm.mapped_column(sqltypes.JSON)
-    evaluators: orm.Mapped[list[str]] = orm.mapped_column(sqltypes.JSON)
+    evaluators: orm.Mapped[dict] = orm.mapped_column(sqltypes.JSON)
```

Comment:
why the change from `list[str]` to `dict`?

Reply:
Becuase it should have been a `dict` from the start. `evaluators` in the old `DIBBS_BASIC` and `DIBBS_ADVANCED` `algo_config`s where just `dict`.

Reply:
Ah thats right, thanks for the reminder I was confusing this with blocking_keys for a min.  Would you mind making this small change though?
```suggestion
    evaluators: orm.Mapped[dict[str,str]] = orm.mapped_column(sqltypes.JSON)
```